src/components/Header/index.js

```diff
@@ -0,0 +1,53 @@
+// https://govuk-elements.herokuapp.com/typography/#typography-headings
+// https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/_typography.scss
+// https://github.com/alphagov/govuk-frontend/blob/master/src/globals/scss/core/_typography.scss
+// https://github.com/alphagov/govuk_elements/blob/master/packages/govuk-elements-sass/public/sass/elements/_elements-typography.scss
+
+import glamorous from "glamorous";
+import React, { createElement } from "react";
+import PropTypes from "prop-types";
+
+import { MEDIA_QUERIES, NTA_LIGHT } from "../../constants/index";
+import { LEVEL_SIZE, FONT_SIZES, LEVEL_TAG } from "../../constants/typography";
+
+const Header = glamorous(({ level, children, ...props }) =>
+  createElement(LEVEL_TAG[level], props, children)
+)(
+  {
+    fontFamily: NTA_LIGHT,
+    fontWeight: "bold",
+    margin: 0
+  },
+  ({ level, size = LEVEL_SIZE[level] }) => ({
+    fontSize: FONT_SIZES[size].mobile.fontSize,
```

Comment:
i would have done mobile first here, e.g. 
_FONT_SIZES[size].fontSize_ and 

for the _MEDIA_QUERIES.LARGESCREEN_ below: 
_FONT_SIZES[size].largescreen.fontSize_